config/sync/field_ui.settings.yml

```diff
@@ -1,3 +1,3 @@
-field_prefix: su_
+field_prefix: cs_
```

Comment:
keep this `su_`

Reply:
![broken record](https://media1.tenor.com/images/8fb344be40606a2ceb421bcafcd945e3/tenor.gif?itemid=15930510)